README.md

```diff
@@ -58,3 +58,32 @@ import FLAnimatedImage from 'react-native-flanimatedimage';
   <FLAnimatedImage style={style} source={source} onLoadEnd={this.onLoadEnd} />
 ...
 ```
+
+## Android
+
+Android uses regular images, but you can still call the component
+
+```js
+import FLAnimatedImage from 'react-native-flanimatedimage';
+
+  <FLAnimatedImage style={style} source={source} onLoadEnd={this.onLoadEnd} />
+...
+```
+
+The module will convert and return a regular image.
+
+By default, GIF is not supported on Android.
+
+You will need to add some optional modules in android/app/build.gradle, depending on the needs of your app.
```

Comment:
actually it can be achieved with fresco gif-animated component, more over fresco (base component) is a part of RN.